third_party/nvfuser/csrc/compute_at_map.h

```diff
@@ -46,6 +46,10 @@ namespace nvfuser {
 //   Map all iteration domains
 //   Always contain root mappings (otherwise they could have been forwarded in
 //   broadcast)
+// IdMappingMode::PERMISSIVE_RESIZE
```

Comment:
Yeah, agreed. I think #2316 by @csarofeen would make this more manageable. For the meantime, I believe it's also good to exercise the indexing map. That would allow us to understand what capability the indexing map would need to have.